tests/boards/intel_s1000_crb/src/spi_flash.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2016 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <flash.h>
+#include <device.h>
+#include <stdio.h>
+
+#define FLASH_TEST_REGION_OFFSET 0x10000
+#define FLASH_SECTOR_SIZE        0x10000
+#define TEST_DATA_BYTE_0         0x4f
+#define TEST_DATA_BYTE_1         0x4a
+#define TEST_DATA_LEN            128
+
+void test_flash(void)
+{
+	struct device *flash_dev;
+	u8_t buf[TEST_DATA_LEN];
+	int i;
+
+	printf("==========================\n");
```

Comment:
s/printf/printk

But you could also use the logging subsystem which is nicer (since you have macros for info/error/warning and debug)

Reply:
done.